enums.h

```diff
@@ -2541,25 +2541,31 @@ typedef enum _setlevels {
 } _setlevels;
 
 typedef enum quest_id {
-	QTYPE_INFRA   = 0x0,
-	QTYPE_BLKM    = 0x1,
-	QTYPE_GARB    = 0x2,
-	QTYPE_ZHAR    = 0x3,
-	QTYPE_VEIL    = 0x4,
-	QTYPE_MOD     = 0x5,
-	QTYPE_BUTCH   = 0x6,
-	QTYPE_BOL     = 0x7,
-	QTYPE_BLIND   = 0x8,
-	QTYPE_BLOOD   = 0x9,
-	QTYPE_ANVIL   = 0xA,
-	QTYPE_WARLRD  = 0xB,
-	QTYPE_KING    = 0xC,
-	QTYPE_PW      = 0xD,
-	QTYPE_BONE    = 0xE,
-	QTYPE_VB      = 0xF,
-	QTYPE_INVALID = -1,

```

Comment:
`Q_INVALID` not present?

Reply:
Most `-1` enums were a workaround for some mess with IDA (can't really remember). It should not be defined as it was generally used for end of arrays and invalid values across the board.